src/vc-api/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@dfinity/internet-identity-vc-api",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc --build",
+    "check": "tsc --noEmit",
+    "postinstall": "npm run build"
+  },
+  "license": "SEE LICENSE IN LICENSE.md",
+  "devDependencies": {
```

Comment:
you need to redeclare those here given that you build from root?

btw. zod and dfinity libs are runtime libs no?

Reply:
oh yes that's right! good catch